docs/sources/permissions/datasource_permissions.md

```diff
@@ -10,11 +10,11 @@ parent = "permissions"
 weight = 4
 +++
 
-# Data Source Permissions
+# Data Source Permissions API
```

Comment:
This page describes concept of data source permissions, and not the API (as opposed to docs/sources/http_api/datasource_permissions.md).